README.md

```diff
@@ -106,6 +106,34 @@ buildctl build ... --exporter=local --exporter-opt output=path/to/output-dir
 buildctl du -v
 ```
 
+#### Running containerized buildkit
+
+buildkit daemon supports communicating with clients over TCP which makes it possible, among other things, to run buildkit within _Docker for Mac_. 
```

Comment:
Not limited to "for Mac"

Reply:
`among other things` but I will rephrase it :)